internal/graph/graph.go

```diff
@@ -332,37 +319,6 @@ func (g *ConnectedObjectGraph) findIngressesWithTargetRewrite(
 					TuplesetRelation: typesystem.DirectRelationReference(target.GetType(), tupleset),
 					Condition:        condition,
 				})
-			} else {
```

Comment:
Could we get an explanation for this change with a reference to a test that demonstrates the reasoning for it?

Reply:
It is adding an edge that should not be added

Reply:
I verified this behavior and it's solid. I [left a comment](https://github.com/openfga/openfga/pull/921/files#r1298896832) to add a few tests just to cover our bases on the original reason why we added this code (albeit we added it in an erroneous way back then).

If we can add those tests then we're good here 👍